code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -111,3 +111,35 @@ Lizard subspecies: ASHWALKERS
 	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,DIGITIGRADE,HAS_FLESH,HAS_BONE)
 	inherent_traits = list(TRAIT_ADVANCEDTOOLUSER,TRAIT_CHUNKYFINGERS,TRAIT_NOBREATH)
 	species_language_holder = /datum/language_holder/lizard/ash
+
+/*
+Lizard subspecies: SILVER SCALED
+*/
+/datum/species/lizard/silverscale
```

Comment:
by the way, because these are subtypes of lizards (but not ash walkers for some reason? are they supposed to not have digitigrade feet?), they're a valid magic mirror race.

I personally think that that's fine (especially if you give them TRAIT_CHUNKYFINGERS back), but hey, just so ya know.

Reply:
why would they be ash walkers they are literally the opposite of them, a secret society royalty

Reply:
fair enough